src/sub-commands/flex-plugin.ts

```diff
@@ -373,4 +388,15 @@ export default class FlexPlugin extends baseCommands.TwilioClientCommand {
   get isJson() {
     return this._flags.json;
   }
+
+  /**
+   * Get the cli plugin configuartion
+   */
+  get pluginsConfig() {
+    mkdirp(join(this.cliRootDir, 'flex'));
```

Comment:
Change to `mkdirp.sync` that's why you are getting the floating exception - `mkdir` is async